docs/Installation-Windows.md

```diff
@@ -53,7 +59,7 @@ You should see `(ml-agents)` prepended on the last line.
 Next, install `tensorflow`. Install this package using `pip` - which is a package management system used to install Python packages. In the same Anaconda Prompt, type in the following command _(make sure you are connected to the internet)_:
 
 ```
-pip install tensorflow
+pip install tensorflow==1.4.0
```

Comment:
In our requirement.txt, we already specified tensorflow==1.4.0 version, why would you specify it here again?

Reply:
Some people copy and paste code from tutorials. If done like that conda would install the latest version of TensorFlow and it will ask for CUDA 9.0 later.

Reply:
Oh I see.